java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/symbol/ConstructorCallSymbolProvider.java

```diff
@@ -58,6 +59,11 @@ public List<SymbolInformation> get(SearchMatch match) throws CoreException {
                     astParser.setResolveBindings(true);
                     CompilationUnit cu = (CompilationUnit) astParser.createAST(null);
                     CustomASTVisitor visitor = new CustomASTVisitor(query, match, QueryLocation.CONSTRUCTOR_CALL);
+                    // Under test, resolveConstructorBinding will return null if there are problems
+                    IProblem[] problems = cu.getProblems();
+                    if (problems != null && problems.length > 0) {
+                        logInfo("KONVEYOR_LOG: " + "Found " + problems.length + " problems while compiling");
```

Comment:
Would it be possible to add more information to this so that there is awareness of where the problems are being found somehow? Maybe some info from the `SearchMatch`

Reply:
The problems appear to be compile errors. For example, one of my test reports: `Preview features enabled at an invalid source release level 21; preview can be enabled only at source level 22`. I’ve updated the code to print the problem messages here. To prevent the log file from growing too large due to repeated errors printed, I limited it to a maximum of 10 lines.